tutorials/algorithms/01_algorithms_introduction.ipynb

```diff
@@ -15,7 +15,7 @@
    "source": [
```

Comment:
<p>Here two drawings of the same circuit are shown but I don't see any problem with the code. There might be some bug.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1370/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Seems to be the case. However, after the latest changes, everything looks fine even with `ReviewNB`.